pwa/app/components/tba/auth/signInWithAppleButton.tsx

```diff
@@ -0,0 +1,17 @@
+import { OAuthProvider } from 'firebase/auth';
+
+import AppleLogo from '~/images/sign-in/apple_sign_in_logo.svg';
+
+import SignInButton from './signInButton';
```

Comment:
Use absolute imports; you can configure VSCode to automatically do them here: https://stackoverflow.com/questions/52432191/auto-import-in-visual-studio-code-only-offering-absolute-path-with-lerna-subpack